docs/system/container-metrics.md

```diff
@@ -0,0 +1,108 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.time` | Counter | `ns` | Total CPU time consumed [1] |
+
+**[1]:** Total CPU time consumed by the specific container.
+<!-- endsemconv -->
+
+<!-- semconv metric.container.cpu.time(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `container.cpu.state` | string | The CPU state for this data point. A container SHOULD be characterized _either_ by data points with no `state` labels, _or only_ data points with `state` labels. | `user`; `kernel` | Opt-In |
+| [`container.id`](../attributes-registry/container.md) | string | Container ID. Usually a UUID, as for example used to [identify Docker containers](https://docs.docker.com/engine/reference/run/#container-identification). The UUID might be abbreviated. | `a3bf90e006b2` | Recommended |
```

Comment:
This should be a resource attribute, right?

Reply:
Ah you mean to make it part of the registry? Good catch, I have moved it.

Reply:
Yes, but it's a _resource_ attribute that is already defined in https://github.com/open-telemetry/semantic-conventions/blob/399978fb27615d7751964b79bc43ad87fe290728/docs/resource/container.md. 

This section is for _data point_ attributes. So I think nothing needs to be defined here

Reply:
Ah ok, so we don't need to define at all the reference to `container.id`, I wasn't aware of this tbh. Is this "rule/guideline" documented somewhere? If not, it would make sense to have it documented.
I have removed it: [30d99e9](https://github.com/open-telemetry/semantic-conventions/pull/282/commits/30d99e9f5abe20e43bb3646baccc8ab348c08718)